README.md

```diff
@@ -0,0 +1,118 @@
+Aubrey-transcription [![Build Status](https://www.travis-ci.com/unt-libraries/aubrey-transcription.svg?branch=master)](https://www.travis-ci.com/unt-libraries/aubrey-transcription)
+====================
+
+
+About
+--------------------
+
+This is a service that returns information about available transcription files
+for a given record. The data is returned as JSON. This service is intended to
+be placed on the same server that the actual transcription files are located on,
+and it must have access to those files, which must be in a pairtree directory
+structure.
+
+
+Requirements
+--------------------
+
+* Python 2.7, 3.4 - 3.6
+* Flask 1.0.2
+* pypairtree 1.1.0
+
+
+Developing/Testing
+--------------------
+
+In order to start developing or run the tests, it is recommended that you set up
+a virtual environment specifically for aubrey-transcription. To do so, download
+virtualenv and then set up a virtual environment with Python 2.7 according to the
+documentation.
+
+One you have a virtual environment, activate it and proceed with the following
+directions to start developing/testing.
+
+1. Clone this repository and move to the project root.
+    ```sh
+        $ git clone git@github.com:unt-libraries/aubrey-transcription.git
+        $ cd aubrey-transcription
+    ```
+
+2. Install the required dependencies.
+    ```sh
+        $ pip install -r test-requirements.txt
+    ```
+
+3. Provide the proper configuration settings. This will be specific to how you
+   are using this application. There are 4 main settings that should be configured
+   properly, either by modifying the "default_settings.py" file in the
+   aubrey_transcription package or by creating a directory called "instance" at the
+   root of the project and putting a new file called "settings.py" there with the
+   following 4 settings:
+
```

Comment:
Do we need to add about the filename pattern setting?

Reply:
I _knew_ there was something I forgot to add!